src/components/Footer/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Wrapper, Divider } from './styles';
+export const Footer: React.FC = () => {
+  const { t } = useTranslation();
+
```

Comment:
you prities and eslint not work, you have 2 ppases here, but after line 3 you dont have empty line. check your code formatting configuration or update if needed the rules